### PR TITLE
Navbar Refactor

### PR DESCRIPTION
**Pull request by siefkenj**
Refactors the navbar and added new styling

### CODE REVIEW

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -18,7 +18,9 @@ class App extends React.Component {
         return (
             <React.Fragment>
                 <AdminHeader />
-                <AdminRoutes />
+                <div className="view-container">
+                    <AdminRoutes />
```
Comment (wisemandenny) [inline frontend/src/App.jsx:22]: does this not work if you do `<AdminRoutes className="view-container" />`?

File: frontend/src/components/header.js [hunk 2]
```diff
@@ -47,7 +49,9 @@ BootstrapNavItem.propTypes = {
 
 export function Header(props) {
     const { routes = [], infoComponents = [] } = props;
-    let match = useRouteMatch("/:mainRoute/:subRoute?");
+    let match = useRouteMatch("/:mainRoute/:subRoute?") || {
+        params: { mainRoute: "tapp" },
```
Comment (wisemandenny) [inline frontend/src/components/header.js:53]: of course the way I wrote it didn't work... good catch

**Review by wisemandenny — COMMENTED** [review]
Looks good. I like the refactor of the header component so that it doesn't need rows and cols.

File: frontend/src/App.jsx (hunk 1 above)
Comment (siefkenj) [reply frontend/src/App.jsx:22]: I believe `AdminRoutes` returns a `React.Fragment`, so it cannot have a class name.

**Pull request merged**